pyo3-ffi/src/pyerrors.rs

```diff
@@ -158,9 +158,9 @@ pub unsafe fn PyUnicodeDecodeError_Create(
     end: Py_ssize_t,
     _reason: *const c_char,
 ) -> *mut PyObject {
-    crate::ffi::PyObject_CallFunction(
+    crate::PyObject_CallFunction(
         PyExc_UnicodeDecodeError,
-        std::ffi::CStr::from_bytes_with_nul(b"sy#nns\0")
+        std::crate::CStr::from_bytes_with_nul(b"sy#nns\0")
```

Comment:
This looks like search+replace gone wrong.

Reply:
I noticed this myself a couple minutes before your review (fixed in the last push) so this is already fixed